src/Orleans/Streams/PubSub/ImplicitStreamSubscriberTable.cs

```diff
@@ -9,10 +10,12 @@ namespace Orleans.Streams
     internal class ImplicitStreamSubscriberTable
     {
         private readonly Dictionary<string, HashSet<int>> table;
+        private readonly List<Tuple<IStreamNamespacePredicate, int>> predicates;
```

Comment:
@jason-bragg As far I understand, it should work as long as `IStreamNamespacePredicate` implementations are serializable. At least it is what I've seen when debugging the tests.